con-base/src/main/java/ba/codecentric/base/service/ImageServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package ba.codecentric.base.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Service
+public class ImageServiceImpl implements ImageService {
+
+    @Value("${image.directory.path}")
+    private String directoryPath;
+
+    @Override
+    public String saveImage(InputStream image, String name) {
```

Comment:
Add `throws IOException` in method signature instead of handling it inside of the method. Add Java docs.